README.md

```diff
@@ -1,15 +1,72 @@
 # loke-http-rpc
 
+## Breaking Changes for v5
+
+Services need to be registered before a request handler can be created.
+
+### Migrating from v4 to v5
+
+- Services now needs to be registered before creating a request handler
+- Added new well know handler ("createWellKnownMetaHandler()") that serves service metadata
+- Exposing well-known URL for uniformity across the system to access service metadata
```

Comment:
Also

- You can now create a request handler for multiple services contained in a separate registry

Reply:
added this now